scripts/benchmark_stats.py

```diff
@@ -89,6 +102,16 @@ def _process_file_data(self):
                         )
                         self._requests[row["Type"]]["total"] += request_count
 
+    @property
+    def pdf_percentile(self) -> Any:
+        if self._pdf_percentile:
+            return int(sum(self._pdf_percentile) / len(self._pdf_percentile))
+        return None
+
+    @property
+    def session_percentile(self) -> int:
+        return int(sum(self._session_percentile) / len(self._session_percentile))
```

Comment:
Same as [the other comment](https://github.com/ONSdigital/eq-survey-runner-benchmark/pull/164/files#r1714766385).

Reply:
Updated 👍